converter/internal/staticconvert/testdata_windows/integrations.river

```diff
@@ -23,18 +23,8 @@ http {
 }
 
 prometheus.exporter.windows "integrations_windows_exporter" {
-	enabled_collectors = ["cpu", "cs", "logical_disk", "net", "os", "service", "system"]
-
-	dfsr {
-		sources_enabled = ["connection", "folder", "volume"]
```

Comment:
This is default coming from https://github.com/grafana/windows_exporter/blob/3391dcec6884e64b0f6face500805c9fbc406b31/collector/dfsr.go#L94